modules/yardstick/DEVNOTES.txt

```diff
@@ -0,0 +1,34 @@
+Building from Ignite sources
+=========================================
+
+1. Create a local clone of Ignite repository
+
+https://git-wip-us.apache.org/repos/asf/ignite
+
+2. Run
+
+mvn clean package -Pyardstick -pl modules/yardstick -am -DskipTests
+
+in the Apache Ignite root directory.
+
+This command will compile the project and also unpack scripts from `yardstick-resources.zip` file to
+`modules/yardstick/target/assembly/bin` directory.
+
+Artifacts can be found in `modules/yardstick/target/assembly` directory.
+
+
+Building from standalone sources
+================================
+
+Run following maven command:
```

Comment:
Thanks, now I see. From the user standpoint it sounds confusing. My suggestion is to leave "modules/yardstick/DEVNOTES.txt" unchanged or have here only those instructions that are needed to build the module from "modules/yardstick" sources.

Special README.txt needs to be generated on the fly for the sources that we add to an Apache binary. The file will encompass "building from standalone sources" instructions and instructions from modified "modules/yardstick/README.txt" that are relevant for the sources in the binary.